incubator/elastic-stack/values.yaml

```diff
@@ -0,0 +1,19 @@
+# Default values for elk.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+kibana:
+  env:
+    ELASTICSEARCH_URL: http://http.default.svc.cluster.local:9200
+
+logstash:
```

Comment:
What's the logic behind these defaults?

Reply:
These are just the defaults for the sub-charts.